### PR TITLE
Add-GRC-Ansible-Automation-everyEvent-mode-on-UI

### PR DESCRIPTION
**Pull request by ChunxiAlexLuo**
Signed-off-by: Chunxi Luo chuluo@redhat.com

https://github.com/stolostron/backlog/issues/23846

### CODE REVIEW

**Comment by ChunxiAlexLuo** [comment]
Screen Shot 2022-07-25 at 2 03 04 PM
Screen Shot 2022-07-25 at 2 03 24 PM
Screen Shot 2022-07-25 at 2 03 32 PM

**Comment by ChunxiAlexLuo** [comment]
@mprahl @JustinKuli Thanks for reviewing the tooltips information text

**Comment by dhaiducek** [comment]
@ChunxiAlexLuo Did you get doc input on the tooltip text?

**Comment by ChunxiAlexLuo** [comment]
> @ChunxiAlexLuo Did you get doc input on the tooltip text?

There is a task but no PR yet, if @dockerymick doesn't have one under working, I can create a Doc PR for review

**Comment by dhaiducek** [comment]
I mean doc squad input for the text here, not for the documentation.

> There is a task but no PR yet, if @dockerymick doesn't have one under working, I can create a Doc PR for review

**Comment by ChunxiAlexLuo** [comment]
> I mean doc squad input for the text here in this PR, not for the documentation.
> 
> > There is a task but no PR yet, if @dockerymick doesn't have one under working, I can create a Doc PR for review

Currently I can't assign the reviewer so thanks for helping to assign  @dockerymick!

**Comment by dhaiducek** [comment]
She's not a member of this org to be assigned as a reviewer, and I'm guessing we've tagged her enough that she's been pinged, but I'll message her directly just in case. 🙂

**Comment by dockerymick** [comment]
@dhaiducek How can I join this org?

File: wizards/PolicyAutomation/PolicyAutomationWizard.tsx
Comment (dockerymick) [inline wizards/PolicyAutomation/PolicyAutomationWizard.tsx:240]: DelayAfterRunSeconds is the minimum seconds before an automation can be restarted on the same cluster. When a policy is violated, the automation runs one time in the delay period, after which it runs every time for a new violation. If the policy is violated multiple times during the delay period and kept in the violated state, the automation runs one time in the delay period and another time after it. The default is 0 seconds and is only applicable for the everyEvent mode.

**Review by dockerymick — COMMENTED** [review]
Left a comment. is it appropriate to say that the automation is being **restarted** or _reinitiated_, _reran_, anything besides retriggering or trigger

**Comment by ChunxiAlexLuo** [comment]
> @dhaiducek How can I join this org?

@jamestalton is the organizer I think

**Comment by ChunxiAlexLuo** [comment]
> Left a comment. is it appropriate to say that the automation is being **restarted** or _reinitiated_, _reran_, anything besides retriggering or trigger

The original design doc used `retriggered` and I think both are appropriate.

**Comment by dockerymick** [comment]
> > Left a comment. is it appropriate to say that the automation is being **restarted** or _reinitiated_, _reran_, anything besides retriggering or trigger
> 
> The original design doc used `retriggered` and I think both are appropriate.

Sounds good, let's avoid using the term "trigger" because it can have a negative interpretation in terms of translation. It is best to say start or initiate

**Review by dockerymick — DISMISSED** [review]
Updates look good from my perspective

**Comment by ChunxiAlexLuo** [comment]
Wait for approval from @jamestalton

File: wizards/PolicyAutomation/PolicyAutomationWizard.tsx
Comment (mprahl) [inline wizards/PolicyAutomation/PolicyAutomationWizard.tsx:237]: @dockerymick could you please review this suggestion?
```suggestion
                        labelHelp="DelayAfterRunSeconds is the minimum seconds before an automation can be restarted on the same cluster. 
                        When a policy is violated, the automation runs one time before the delay period.
                        If the policy is violated multiple times during the delay period and kept in the violated state, 
                        the automation runs one time after the delay period. 
                        The default is 0 seconds and is only applicable for the everyEvent mode."
```

Comment (dockerymick) [reply wizards/PolicyAutomation/PolicyAutomationWizard.tsx]: This suggestion looks good to me! @mprahl

**Review by KevinFCormier — APPROVED** [review]
Looks good. Thanks!

**Pull request merged**
